core/jazz_logs/index.js

```diff
@@ -183,37 +185,38 @@ module.exports.handler = (event, context, cb) => {
     return cb(JSON.stringify(errorHandler.throwInternalServerError("Exception occured while processing the request : " + JSON.stringify(e))));
   }
 
-  function setRequestBody(category, type, querys, startTime, endTime, size, page) {
+  function setRequestBody(category, querys, filterQuerys, startTime, endTime, size, page) {
     var index = {
       "index": category,
-      "type": type,
       "ignore_unavailable": true
     };
 
+
     var params = {
       "size": size,
       "from": page,
       "sort": [{
         "timestamp": {
-          "order": "desc"
+          "order": "desc",
+          "unmapped_type": "boolean"
```

Comment:
What does this do?

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/6.3/search-request-sort.html#_ignoring_unmapped_fields

Reply:
@SiniWilson what happens if unmapped_type is not set or is set to a different data type such as `date` or `long`?

Reply:
This is not needed at all since timestamp is already mapped to type - `date` for this index. 

See https://github.com/tmobile/jazz-installer/blob/master/installer/terraform/scripts/configureESEndpoint.sh#L20 &  https://github.com/tmobile/jazz/blob/master/core/jazz_es-kinesis-log-streamer/_ES/apilogs.json